server/webserver.go

```diff
@@ -1019,13 +1019,15 @@ func (s *LivepeerServer) StartWebserver() {
 			if err != nil {
 				glog.Infof("Ping failed for %v", bootID)
 				result[fmt.Sprintf("%v,%v", bootID, bootAddr)] = false
+				w.WriteHeader(http.StatusBadRequest)
```

Comment:
Looks OK. Why does it get a HTTP error code here, when none of the other endpoints have one?

Reply:
Gonna use the status code in a monitor script to restart the node if pinging bootnodes fail.